### PR TITLE
bulk is an access matter, form shouldn't be specific to data, machine…

### PR DESCRIPTION
**Pull request by mlinksva**
- move bulk should to access
- machine-readibility is implied by format must be processable by a floss tool and is otherwise vague
- data should not be called out in particular. bulk access and convenient format issues equally apply to other sorts of works, eg streaming-only access to video is bad
- preferred form for modification also covers read/parsibility, and is a should not clear to me appropriate for open definition to completely rule out eg pdfs and lossy files as Rob Myers said in https://lists.okfn.org/pipermail/od-discuss/2015-April/001339.html

### CODE REVIEW

File: source/open-definition-2.1-dev.markdown [hunk 1]
```diff
@@ -42,8 +42,7 @@ accompany the **work**.
 
 The **work** *must* be provided in an open format. An open format is
 one which places no restrictions, monetary or otherwise, upon its use and can be fully processed
-with at least one free/libre/open-source software tool. Data *must* be machine-readable and 
-*should* be provided in bulk.
+with at least one free/libre/open-source software tool. The **work** *should* be provided in the form preferred for making modifications to it.
```
Comment (wolftune) [inline source/open-definition-2.1-dev.markdown]: I don't understand why work should be bold in this second use. Otherwise looks good to me

Comment (Stephen-Gates) [inline source/open-definition-2.1-dev.markdown]: I like the changes. I'm not sure about "preferred". I think we would either:
- need to separately provide some guidance or examples on preferred formats as peoples preferences will vary.
- change "preferred" to something that drives people to at least 3 star open data publishing.

**Comment by mlinksva** [comment]
@wolftune I removed bold of 2nd mentions of work in paragraph.

@Stephen-Gates preferred format for modification ought drive toward 3 star (ie 2nd star requirement) publishing. FWIW the terminology is taken from the GPL's " preferred form of the work for making modifications"; I haven't seen a better statement which applies to all kinds of knowledge, but that doesn't mean it can't be done!

**Comment by wolftune** [comment]
@Stephen-Gates echoing @mlinksva the "preferred" can be interpreted such that if any given party has a form they prefer, it is that form which must be distributed to others. This provides the most flexible and yet strong definition. People can argue about what should be preferred. The real violation is preferring one form for yourself but giving other only some other form.

All that said, I want to see separate references as to what forms of various media are generally recommended for Open works and what defines an Open Format. Those details don't strictly replace the "preferred form" line because even if the format is correct, a compressed / minified / compiled version of the work is not the form preferred for modifying.

**Comment by Stephen-Gates** [comment]
I agree with "*should* be provided in bulk" but see [this conversation](https://discuss.okfn.org/t/penalty-for-no-bulk-data-not-appropriate-for-realtime-or-big-data/294?u=stephen) with @rgrp in the OK Forum where he has a stronger view on the provision of bulk data.

**Comment by wolftune** [comment]
@Stephen-Gates we still need to keep from conflating concerns that are specific to bulk data from things that apply to all Open anything. Having specifications for data in particular seems okay to me in principle, depending on details; as long as it is clearly that.

**Comment by hlainchb** [comment]
@wolftune this seems to fly in the face of what @mlinksva said to begin with in this thread, which I tend to agree with.  I don't see why data is special when it comes to bulk.  I may be misunderstanding here.  Would specific examples of where bulk would not be appropriate.  Assuming we can sufficiently define bulk. :)

**Comment by mlinksva** [comment]
I agree data isn't special when it comes to bulk. I realized that "as a whole" already covers all media types/subjects/domains, so the world bulk no longer appears.

I'm going to go ahead and merge this now as it seems like progress, though in the end the diff is slight.

If format-specific examples are needed @Stephen-Gates or whoever is interested should open up a new issue or submit a new pull request addressing. Possibly they should go in the OFD document Stephen has primarily been working on rather than the OD?

**Comment by Stephen-Gates** [comment]
@mlinksva I'm happy with the changes and dealing with format on OFD. Thanks

**Pull request merged**